plugins/main/public/react-services/reporting.js

```diff
@@ -94,12 +94,24 @@ export class ReportingService {
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiButton
-              onClick={() =>
-                window.open(
-                  getHttp().basePath.prepend(`/reports/${filename}`),
-                  '_blank',
-                )
-              }
+              onClick={() => {
+                // Open the new tab immediately
+                const windowReference = window.open('', '_blank');
+
+                // Verify if the new tab was locked
+                if (windowReference) {
+                  // Get the URL (in this case, it is obtained directly, but it can be asynchronous)
+                  const url = getHttp().basePath.prepend(
+                    `/reports/${filename}`,
+                  );
+
+                  // Then assign the URL to the new tab
+                  windowReference.location = url;
+                } else {
+                  // If the tab is locked, it displays an alert to the user.
+                  alert('The tab was blocked. Please enable pop-up windows.');
+                }
+              }}
```

Comment:
**thought**: I see you changed to `window.open`. For this case, we could try a cleaner solution using a link instead of manage the access to the report with a `onClick`. Could you try it?

In the case we implement the proposed solution in the `onClick`, there are more usages of `window.open` that could be failing in Safari too. We should review them.